src/ng/compile.js

```diff
@@ -2545,9 +2545,19 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
         lastValue,
         parentGet, parentSet, compare;
 
+        if ((attrName in attrs) && !hasOwnProperty.call(attrs, attrName)) {
```

Comment:
Em... Tried to run the test and seems all good. I've updated the commit. Thanks.